src/super_gradients/training/utils/predict/prediction_results.py

```diff
@@ -129,6 +129,22 @@ def draw(self, box_thickness: int = 2, show_confidence: bool = True, color_mappi
                 y2=int(self.prediction.bboxes_xyxy[pred_i, 3]),
             )
 
+        if plot_targets:
+            target_image = self.image.copy()
+            for target_idx in range(len(self.prediction.target_bboxes_xyxy)):
+                class_id = int(self.prediction.target_labels[target_idx])
+                target_image = draw_bbox(
+                    image=target_image,
+                    title=f"{self.class_names[class_id]}_GT",
+                    color=color_mapping[class_id],
+                    box_thickness=box_thickness * 3,
```

Comment:
no need to make it thicker if it is side by side

Reply:
dropped.